### ISSUE TITLE
Can't MakeAuth. Phone is banned.

### ISSUE THREAD

**Issue by WestSide072** [sochix/TLSharp#475]
```
        public async void Auth(string phone)
        {
            FileSessionStore store = new FileSessionStore();
            //TelegramClient client = new TelegramClient(98996, "fea153bd8aefd5e1ce908bfe991832f6", store, null); 
            TelegramClient client = new TelegramClient(98996, "fea153bd8aefd5e1ce908bfe991832f6", store, null
            , (address, port) =>
              {
                  var tcp = new TcpClient();
                  tcp.Connect("149.154.167.50", 443);
                  return tcp;
              });
            await client.ConnectAsync(false);

            string hash = await client.SendCodeRequestAsync(phone);

            string code = "change_me";
            await client.MakeAuthAsync(phone, hash, code); // banned phone after this line  
        }
```

**Comment by ali-kamali** [comment]
#439

**Comment by WestSide072** [comment]
Problem at ip connect? I will not write at support for every registration...  
// sorry for my English

**Comment by ali-kamali** [comment]
you try too many times to login.

**Comment by WestSide072** [comment]
Login in web.telegram.org and next with TLSharp.

**Comment by jokerbirot** [comment]
I'm facing something similar. After some time i performed Authenticate, when performing: 
var hash = await client.SendCodeRequestAsync(XXXXX);
i receive an error like this: 
Flood prevention. Telegram now requires your program to do requests again only after 60044 seconds have passed (TimeToWait property). 
So i'm blocked for more than 60k seconds

I tried to use the other IP (149.154.167.40) that in my telegram development tools is signed as "test" but in that case i receive APP_ID_INVALID.

There's no other way than "wait" until that time has expired? Moreover, how could i overcome this issue? i mean, how can i avoid requiring new "codes" for authentication and being authenticated even if i re-lauch my application?

**Comment by knocte** [comment]
Duplicate of https://github.com/sochix/TLSharp/issues/561
